### PR TITLE
Feature/student list

### PR DESCRIPTION
**Pull request by DenGodovanets**
Something classes are empty, so when you choose empty class, you cant to see students list.

### CODE REVIEW

File: src/app/admin-panel/students-list/students-list.component.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          <span>{{notActive.className}}</span>
+          <span>-</span>
+          <span>{{notActive.classYear}}рік</span>
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+  </div>
+</div>
+<div [ngClass]="'students'">
+    <mat-list *ngIf="studentList">
```
Comment (dmitryy90) [inline src/app/admin-panel/students-list/students-list.component.html]: indentations

File: src/app/admin-panel/students-list/students-list.component.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.select {
+    display: flex;
```
Comment (dmitryy90) [inline src/app/admin-panel/students-list/students-list.component.scss]: indentations

File: src/app/admin-panel/students-list/students-list.component.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <mat-option *ngFor="let notActive of notActiveClass" [value]="notActive.id">
+          <span>{{notActive.className}}</span>
+          <span>-</span>
+          <span>{{notActive.classYear}}рік</span>
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+  </div>
+</div>
+<div [ngClass]="'students'">
```
Comment (dmitryy90) [inline src/app/admin-panel/students-list/students-list.component.html]: why dont you use html class?

File: src/app/services/students.service.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class StudentsService {
+
+private readonly url: string = 'http://35.228.220.5:8080';
```
Comment (dmitryy90) [inline src/app/services/students.service.ts]: update branch and remove this line

File: src/styles.scss [hunk 5]
```diff
@@ -1,4 +1,6 @@
+@import '../node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css';
```
Comment (dmitryy90) [inline src/styles.scss]: it must be declared in angular.json

File: src/app/admin-panel/students-list/students-list.component.html (hunk 1 above)
Comment (DenGodovanets) [reply src/app/admin-panel/students-list/students-list.component.html]: done

File: src/app/admin-panel/students-list/students-list.component.html (hunk 3 above)
Comment (DenGodovanets) [reply src/app/admin-panel/students-list/students-list.component.html]: has remade

File: src/app/services/students.service.ts (hunk 4 above)
Comment (DenGodovanets) [reply src/app/services/students.service.ts]: has remade

File: src/styles.scss (hunk 5 above)
Comment (DenGodovanets) [reply src/styles.scss]: its only for I can to see how will be look

File: yarn.lock [hunk 6]
```diff
@@ -192,6 +192,15 @@
   dependencies:
     tslib "^1.7.1"
 
+<<<<<<< HEAD
+"@angular/material@angular/material2-builds":
```
Comment (dmitryy90) [inline yarn.lock]: generate new lock file

Comment (DenGodovanets) [reply yarn.lock]: done

**Pull request merged**